examples/_client.rb.example

```diff
@@ -16,7 +16,7 @@ require 'pry'
 
 # This setting allows for use of the shortened name of the resources, like OneviewSDK::<Resource_name>,
   # when referring to OneviewSDK::<API_version>::Synergy::<Resource_name>
-# Versions supported: [200, 300, 500, 600, 800, 1000, 1200, 1600, 1800, 2000]
+# Versions supported: [200, 300, 500, 600, 800, 1000, 1200, 1600, 1800, 2000, 2200]
```

Comment:
we are not supporting all these versions

Reply:
we are having support, its just we have modified logic for defaulting API version